doc: document kubernetes private locations configuration and deployment (#122)
gatling/frontline-cloud-doc: content/reference/admin/private_locations/configuration/includes/control-plane.aws.md

      # Subnets
      subnets = ["subnet-a", "subnet-b"]
      # Profile name (optional)
      profile-name = ""
      # IAM Instance profile (optional)
      iam-instance-profile = ""
      # Custom tags
      tags {
       # ExampleKey = ExampleValue
      }
    }
  ]
}
```

{{< alert tip >}}

> AWS private locations require the control plane to have access to credential set on provider chain.
> 
> See [AWS documentation of the Default Credential Provider Chain](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default)

Reviewer: ```suggestion
AWS private locations require the control plane to have access to AWS credentials from the default credential provider chain.

See [the AWS documentation for the Default Credential Provider Chain](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default).
```
